integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMonitoringExporterSideCar.java

```diff
@@ -400,7 +401,9 @@ public void tearDownAll() {
           .withFailMessage("uninstallNginx() did not return true")
           .isTrue();
     }
-
+    shutdownDomain(domain1Namespace, domain1Uid);
```

Comment:
Hi Marina, It doesn't help to shutdown the domains in the tearDown. These domains will be shutdown anyway once the IT class is done. We want to shut it down after the test method uses the domain and other test methods no longer needs it.

Reply:
fixed and rerun in https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/7091